pkg/kubelet/kubelet_getters.go

```diff
@@ -48,6 +48,13 @@ func (kl *Kubelet) getRootDir() string {
 	return kl.rootDirectory
 }
 
+// getPodLogsDir returns the full path to the directory that kubelet can use
+// to store pod's log files. This defaults to /var/log/pods if not specified
+// otherwise in the config file.
+func (kl *Kubelet) getPodLogsDir() string {
+	return kl.podLogsDirectory
```

Comment:
What's the difference between PodLogs*Root*Directory and podLogsDirectory? Do we really need to use 'Root' in the name?

Reply:
Updated